### PR TITLE
[JBPM-9983] Allow to define number of Retries for WIH exception handl…

### PR DESCRIPTION
**Pull request by abhijithumbe**
…ing strategy

**JIRA**: _(https://issues.redhat.com/browse/JBPM-9983)_

### CODE REVIEW

**Comment by afalhambra** [comment]
Jenkins retest this

**Comment by afalhambra** [comment]
@abhijithumbe - just FYI, current active branch for new development in this repo is 7.x branch and not the main.

So you need to close this PR and open a new one against the 7.x branch

**Comment by abhijithumbe** [comment]
@afalhambra okay I will close this PR and will open new against 7.x branch

File: kie-api/src/main/java/org/kie/api/runtime/process/ProcessWorkItemHandlerException.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
     private static final long serialVersionUID = 328927887939759492L;
 
-    private static final int MAX_NUMBER_OF_RETRIES = 3;
+    private static int MAX_NUMBER_OF_RETRIES = Integer.parseInt(System.getProperty("org.jbpm.exception.handling_strategy.retry.count", "3"));
```
Comment (afalhambra) [inline kie-api/src/main/java/org/kie/api/runtime/process/ProcessWorkItemHandlerException.java:23]: it should be `final`

**Review by afalhambra — CHANGES_REQUESTED** [review]

**Pull request closed without merging**